service/templates/authorize.html

```diff
@@ -3,33 +3,37 @@
 {% block content %}
 <main class="s-form-page">
   <form action="" method="post" class="s-form s-form--login">
-    <h1> Tapis Authorization - Tenant {{ tenant_id }}</h1>
-    <p> You are logged in as user {{ username }}. The client application {{ client_display_name }} is requesting a Tapis
-        access token on your behalf.</p>
+    <h1>Tapis Authorization - Tenant {{ tenant_id }}</h1>
+    <p>You are logged in as user <var>{{ username }}</var>. The client application <var>{{ client_display_name }}</var> requests a Tapis access token on your behalf.</p>
 
     {% if error %}
     <ul>
         <li>Error: {{ error }}</li>
     </ul>
     {% endif %}
 
-     <div class="has-required">
-        <label>
-          {% if device_login %}
-          <label for="ttl">Time for token to live in days (default: 30)</label>
-          <input type="text" id="ttl" name="ttl">
-          {% endif %}
```

Comment:
**Before.** This is its own field, not part of the Approve field, so it should have its own `<div>`. Nesting a `<label>` within another (as this old code did)... does not compute.